### ISSUE TITLE
Document full dependencies 

### ISSUE THREAD

**Issue by coretemp** [getsentry/sentry-elixir#282]
By specifying the below you are saying that for all combinations of software on which Elixir runs, your software runs. However, we already know that this isn't the case, because of existing (unresolved) bug reports related to parallel compilation; bugs your team has *not* addressed in an efficient or effective manner, I should add.

```
defp application do
  [applications: [:sentry, :logger]]
end

defp deps do
  [{:sentry, "~> 6.3"}]
end
```
So, before I will actually try to debug anything, I would like to hear from you -- the vendor -- what the dependencies actually are to make this work on generic Linux. 

Preferably I would like to see this fully automated in a script of some kind named `check_dependencies`. This script would check for the operating system, libraries on which it depends (which you have done partially, I believe already), OTP versions, etc. The script should output either an exit status of 0 signifying that a subsequent compile is *guaranteed* to work or an exit status of 1 when the system configuration is not OK with a list of errors of what's wrong.

**Comment by jeregrine** [comment]
In the future please link to the issues and errors that you are having. I understand you are having trouble and this is a very annoying bug. You must understand how hard it is to fix an issue when its not easy to reproduce your self. The issue as presented is not a productive way to interact with any human being.

It turns out that this is a bug in the Elixir Compiler and is being fixed for elixir 1.7. https://github.com/elixir-lang/elixir/issues/7699 

The issue is with how deps are being reported to sentry, the only fix at the moment would remove dep's from being reported at all until mix is fixed. We're discussing possible options and should have a fix out soonish.

**Comment by jeregrine** [comment]
This should temporarily fix the issue.
